lib/openstudio-standards/utilities/assertion.rb

```diff
@@ -62,6 +62,21 @@ def get_additional_property_as_boolean(component, key, default = false)
   return value
 end
 
+# PRM get an additional property from an OpenStudio object as a double,
+# if no such additional property, then return default value.
+# @param component [OpenStudio object] the component to get the additional property from
+# @param key [String] key string
+# @param default [Boolean] the default to return when there is no matching key
```

Comment:
We should probably add the documentation for the return.

Reply:
Updated